addons/dockable_container/dockable_container.gd

```diff
@@ -80,6 +81,8 @@ func _ready() -> void:
 	_split_container.name = "_split_container"
 	_split_container.mouse_filter = MOUSE_FILTER_PASS
 	_panel_container.add_child(_split_container)
+	_windows_container.name = "_windows_container"
+	get_parent().call_deferred("add_child", _windows_container)
```

Comment:
If this script creates `_windows_container`, it should also destroy it, likely in `_exit_tree`.
By the way, can't `_windows_container` be a direct child of `DockableContainer` instead of a sibling?